src/index.js

```diff
@@ -0,0 +1,163 @@
+import './style.css';
+import Item from './Item';
+import Storage from './Storage';
+
+window.storage = 'default';
+if (!localStorage.getItem('projects')) {
+  localStorage.setItem('projects', '');
+}
+
+function render() {
```

Comment:
* [ ] It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/))
- let us redefine the method using` ES6` syntax